script.js

```diff
@@ -0,0 +1,62 @@
+function capitalize(string) {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+function reverseString(string) {
+  return string.split("").reverse().join("");
+}
+
+const calculator = {
+  add: function (a, b) {
+    return a + b;
+  },
+  subtract: function (a, b) {
+    return a - b;
+  },
+  divide: function (a, b) {
+    return a / b;
+  },
+  multiply: function (a, b) {
+    return a * b;
+  },
+};
+
+function caesarCipher(string, shift) {
```

Comment:
Set a default shift value for when it is not specified

```suggestion
function caesarCipher(string, shift = 1) {
```